### PR TITLE
📝 Update docs 

### PR DESCRIPTION
**Pull request by alejsdev**
📝 Update docs

### CODE REVIEW

File: docs/tutorial/connect/create-connected-rows.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Let's start by creating two teams:
 
 ///
 
-This would hopefully look already familiar.
+This would hopefully looks already familiar.
```
Comment (tiangolo) [inline docs/tutorial/connect/create-connected-rows.md]: This change would apply if the phrase was:

> This hopefully looks

But having the `would` there then it should be:

> This would hopefully look

...can you please revert this change?

**Review by tiangolo — COMMENTED** [review]
Thank you! Good catches! :nerd_face: 

Just one change to revert and it's ready.

File: docs/tutorial/connect/create-connected-rows.md (hunk 1 above)
Comment (alejsdev) [reply docs/tutorial/connect/create-connected-rows.md]: Opsie, my bad :smile: done

**Review by tiangolo — APPROVED** [review]

**Comment by tiangolo** [comment]
Nice, thank you @alejsdev! :rice: :cake: :milk_glass:
